go-apps/meep-app-enablement/server/app-support/app-support.go

```diff
@@ -156,6 +165,33 @@ func Stop() (err error) {
 	return nil
 }
 
+func repopulateAppTerminationNotificationSubscriptionMap(key string, jsonInfo string, userData interface{}) error {
```

Comment:
Is this code required? Was it meant to handle cases where the pod restarts?
NOTE: on a pod restart the init function will flush the "app-enablement" redis entries, including the subscriptions. Is there any case where this code will be invoked on a list that is not empty?

Reply:
added a note that explains ReInit not doing anything irght now because of Flush in Init(), but if sbi is added, would become meaningful